docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -157,15 +157,13 @@ dune --deploy ./build/hello hello
 
 > 👀 **Code Permission**
 > 
-> By default, DUNE adds the `eosio.code` permission to an account when you deploy a contract to it. This allows the
-> contract to be able to trigger inline actions on other smart contracts.
+> By default, DUNE adds the `eosio.code` permission to an account when you deploy a contract to it. This allows the contract to trigger inline actions on other smart contracts.
 
 ### Interacting with your contract
 
-To interact with your contract you will send a transaction on your local EOS node. Transactions on EOS are made up of 
-`actions`, so we will send a single action to your contract.
+To interact with your contract send a transaction on your local EOS node. Transactions on EOS are made of `actions`. Send a single action to your contract.
```

Comment:
The way this reads is hard to grok. 

It's telling me conflicting things now:
"I'm supposed to interact with my contract by sending a transaction, but they are made up of actions, and I should send a single transaction? I thought I was supposed to send a transaction?"

The original suffered from this as well, so let's clarify. Maybe something along the lines of: 
"To interact with your contract you need to send a transaction to the blockchain. Transactions are made up of multiple `actions`. You can send a transaction with a single action using the `--send-action` command."

Reply:
How about:
Send a transaction on your local EOS node to the blockchain to interact with your smart contract. A transaction contains multiple actions. You ...